backend/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "jsmp-backend",
+  "version": "1.0.0",
+  "description": "javascript mentoring program 2020",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server/index.ts",
+    "build": "tsc",
+    "test": "jest",
+    "lint": "eslint server --ext 'js,ts'"
```

Comment:
It doesn't work. Could you try this? `"lint": "eslint server/**/*.{js,ts}"`
And also for the future: `"lint:fix": "eslint --fix server/**/*{.ts,.js}"`